kustomize/base/7.1.0/ingress/ingress.yaml

```diff
@@ -93,14 +93,42 @@ spec:
                 name: ig
                 port:
                   number: 80
-            path: /jwkms
+            path: /jwksproxy
             pathType: Prefix
+  tls:
+    - hosts:
+        - $(IG_FQDN)
+      secretName: sslcert
+---
+# Ingress for the Test Trusted Directory hosted by the gateway.
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream: "true"
+    nginx.ingress.kubernetes.io/auth-tls-secret: $(NAMESPACE)/obri-ca
+    nginx.ingress.kubernetes.io/auth-tls-verify-client: optional_no_ca
+    nginx.ingress.kubernetes.io/http2-max-field-size: 16k
+    nginx.ingress.kubernetes.io/http2-max-header-size: 128k
+    nginx.ingress.kubernetes.io/proxy-body-size: 64m
+    nginx.ingress.kubernetes.io/proxy-buffer-size: 128k
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/proxy-buffers: 4 256k
+    nginx.ingress.kubernetes.io/proxy-busy-buffers_size: 256k
+    nginx.ingress.kubernetes.io/error-log-level: "debug"
+  name: test-trusted-directory
+spec:
+  rules:
+    - host: $(IG_FQDN)
```

Comment:
Hosting the directory on the IG domain, we could create another DNS entry to completely separate the test trusted directory.

e.g. test-directory.sapig.$env.forgerock.financial